static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -105,9 +109,6 @@ export default function FeedbackListPage({}: Props) {
 }
 
 const LayoutGrid = styled('div')`
-  background: ${p => p.theme.background};
-  overflow: hidden;
```

Comment:
afaict this is not needed and causes the focus rings to be cut off on the page filters

Reply:
focus rings arn't cutoff anywhere on /feedback

/replay does have that problem though, hard to fix because of some layers that the Trace tab needs :(